internal/store/statefulset_test.go

```diff
@@ -63,6 +63,7 @@ func TestStatefulSetStore(t *testing.T) {
 				# HELP kube_statefulset_created [STABLE] Unix creation timestamp
 				# HELP kube_statefulset_labels [STABLE] Kubernetes labels converted to Prometheus labels.
 				# HELP kube_statefulset_metadata_generation [STABLE] Sequence number representing a specific generation of the desired state for the StatefulSet.
+                # HELP kube_statefulset_persistent_volume_claim_retention_policy Count of retention policy for StatefulSet template PVCs
```

Comment:
indentation here seems to be off

Reply:
thx, tabs vs spaces is the gift that keeps on giving...